packages/nextjs/app/_components/CollateralGraph.tsx

```diff
@@ -46,68 +46,54 @@ const CollateralGraph = () => {
     receiptData: true,
   });
 
-  const { data: priceEvents } = useScaffoldEventHistory({
-    contractName: "StableCoinEngine",
-    eventName: "PriceUpdated",
-    fromBlock: 0n,
-    watch: true,
-    blockData: true,
-    transactionData: true,
-    receiptData: true,
-  });
-  const combinedEvents = [
-    ...(addEvents || []),
-    ...(withdrawEvents || []),
-    ...(transferEvents || []),
-    ...(priceEvents || []),
-  ];
-  const sortedEvents = combinedEvents.sort((a, b) => Number(a.blockNumber - b.blockNumber));
```

Comment:
We'll still want the `priceEvents` here so the graph updates when the price changes